platemate/food/views.py

```diff
@@ -136,6 +143,29 @@ def fe_day(request, day):
     except ValueError:
         return HttpResponseNotFound("Invalid date")
 
+@csrf_exempt
+def api_submission_statuses(request):
+    if request.method == 'POST':
```

Comment:
I think this should be GET? Also we should be authenticating both API endpoints, yes? Perhaps move the auth code into a separate method and just add a guard at the top of both of these that calls it?